clang/test/Sema/assertion-crash.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 -fsyntax-only -std=c++11 %s -verify
+// RUN: %clang_cc1 -fsyntax-only -std=c++23 %s -verify
+
+// expected-no-diagnostics
+
+struct A {};
+using CA = const A;
+
+struct S1 : CA {
```

Comment:
These examples: https://godbolt.org/z/8T3643496 don't crash clang and so don't really test the original bug. Please add the example from the bug report that does crash: https://godbolt.org/z/4naMG1q6r so we have a proper regression test.

Reply:
In the `godbolt` links, the compilers seem to be two different versions of `clang`, one being a `trunk` and other is an `assertion trunk`?